controllers/articles.controllers.js

```diff
@@ -47,3 +48,17 @@ exports.patchArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postArticleComment = (req, res, next) => {
+  const articleID = req.params.article_id;
+  const username = req.body.username;
+  const comment = req.body.comment;
+
+  sendArticleComment(articleID, username, comment)
+    .then((response) => {
+      res.status(200).send({ comment: response });
```

Comment:
why not to change the variable name in the the .then to comment so you can send only {comment} without renaming? No changes in functionality, just some syntactic suuuuugar